node/tessel-export.js

```diff
@@ -85,6 +99,64 @@ Tessel.prototype.close = function() {
   }, this);
 };
 
+Tessel.prototype.pwmFrequency = function(frequency, cb) {
+  if (frequency < Tessel.pwmMinFrequency || frequency > Tessel.pwmMaxFrequency) {
+    throw new RangeError(`pwmFrequency value must be between ${Tessel.pwmMinFrequency} and ${Tessel.pwmMaxFrequency}`);
+  }
+
+  var results = determineDutyCycleAndPrescalar(frequency);
+
+  Tessel.pwmPeriod = results.period;
+  Tessel.pwmPrescalarIndex = results.prescalarIndex;
+
+  // We are currently only using TCC Bank 0
+  // This may be expanded in the future to enable PWM on more pins
+  const TCC_ID = 0;
+
+  var packet = new Buffer(4);
+  // Write the command id first
+  packet.writeUInt8(CMD.PWM_PERIOD, 0);
+  // Write our prescalar to the top 4 bits and TCC id to the bottom 4 bits
+  packet.writeUInt8((Tessel.pwmPrescalarIndex << 4) | TCC_ID, 1);
+  // Write our period (16 bits)
+  packet.writeUInt16BE(Tessel.pwmPeriod, 2);
+
+  // Send the packet off to the samd21
+  // on the first available port object (regardless of name)
+  this.port[["A", "B"].find(name => this.ports[name] !== null)].sock.write(packet, cb);
```

Comment:
I apologize for using double quotes when I wrote this suggestion, it's causing travis failures in jscs tests